cpp/ql/lib/semmle/code/cpp/Specifier.qll

```diff
@@ -38,7 +38,7 @@ class FunctionSpecifier extends Specifier {
 
 /**
  * A C/C++ storage class specifier: `auto`, `register`, `static`, `extern`,
- * or `mutable".
+ * or `mutable`.
```

Comment:
Good spot.

Reply:
Took me a while to realize why it was not highlighting in VS Code 😄